packages/mdc-fab/README.md

```diff
@@ -39,68 +37,71 @@ npm install --save @material/fab
 
 ## Usage
 
-The demonstrations use the [Material Design Icon Font](https://design.google.com/icons/).
-You may include this to use them as shown or use any other icon method you wish.
+### Load Material Icons
 
-### Default
+We recommend you load [Material Icons]((https://design.google.com/icons/)) from Google Fonts
 
 ```html
-<button class="mdc-fab material-icons" aria-label="Favorite">
-  <span class="mdc-fab__icon">
-    favorite
-  </span>
-</button>
+<head>
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
 ```
 
-### Mini
+### HTML Structure
 
 ```html
-<button class="mdc-fab mdc-fab--mini material-icons" aria-label="Favorite">
+<button class="mdc-fab material-icons" aria-label="Foo label">
   <span class="mdc-fab__icon">
-    favorite
+    foo_icon
   </span>
 </button>
 ```
 
-### Absolutely positioned
+### CSS Classes
```

Comment:
Is it normal for this to be as far removed from the Sass sections of the readme as it is? (e.g. should we consider moving the Absolutely Positioned and Adding MDC Ripple sections further down below the Sass stuff?)

Reply:
I think users will use HTML and CSS. Then they will probably also want to position their FAB. They will probably also want to add a Ripple, though maybe not if JS is too heavy. The least likely thing they want to do is customize the FAB's color (least likely after the positioning and ripple). So I've sorted the features to the top which I think most users will want.